src/modules/universe/RadixNodeDiscoveryHardcoded.ts

```diff
@@ -0,0 +1,33 @@
+import axios from 'axios'
+
+import RadixNodeDiscovery from './RadixNodeDiscovery'
+
+import { RadixSerializer } from '../atommodel'
+import { RadixNode, RadixNodeInfo } from '../..'
+
+/**
+ * Radix node discovery from a fixed list
+ */
+export default class RadixNodeDiscoveryHardcoded implements RadixNodeDiscovery {
+    
+    /**
+     * Creates an instance of radix node discovery from seed.
+     * @param bootstrapNode Full address to the rpc endpoint of a Radix node in the universe
+     */
+    constructor(
+        readonly bootstrapNodes: string[],
+        ) {}
+
+    public async loadNodes() {
+        return Promise.all(this.bootstrapNodes.map(async (address) => {
+            const response = await axios.get(`http://${address}/api/network`)
```

Comment:
Off-topic: as I read more and more about Vue.js I keep seeing axios quite frequently. Is it the de facto standard HTTP  client to use in JS based apps in general? Is it legacy in our code base? Should I switch to it in the ABT front end?

Reply:
It's the go-to one for me because it gives you full-featured promise based cross-platform interface, sure, but in this case I don't have any frameworks like Vue providing me with a default solution. I wouldn't switch away from vue-resource unless there's a specific reason.